src/server_manager/electron_app/util.ts

```diff
@@ -25,3 +25,9 @@ import {URL} from 'url';
 export function redactManagerUrl(s: string) {
   return new URL(s).pathname.split('/').slice(2).join('/');
 }
+
+/** Represents an additional trust anchor for a single host. */
+export interface HostAnchor {
```

Comment:
Can you move this to some `trust_certificate.ts` file? We should refrain from files named `util`. That's meaningless.

Perhaps put it under `src/server_manager/common/trust_certificate.ts`, so we can use in the web_app too.

Reply:
BTW, the `trust-certificate` string could also live in that file.

Reply:
.d.ts files can't depend on .ts files AFAICT.  I've moved this to `trust_certificate.d.ts`, but that can only hold types, not actual values like the string name.

Reply:
This claims it's possible: https://stackoverflow.com/a/36183083/1234111

Reply:
Better reference: https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#dependencies-on-modules

Reply:
This only supports "absolute" imports (i.e. imports from node_modules), not "path-relative" imports, presumably because the .d.ts file could be moved before it is used.